base_layer/wallet_ffi/src/lib.rs

```diff
@@ -4535,10 +4537,14 @@ pub unsafe extern "C" fn wallet_import_utxo(
         TariScript::default(),
         ExecutionStack::default(),
         &(*source_public_key).clone(),
-        // WARNING, this might be a problem in the future when importing anything else than a default features UTXO,
-        // the FFI function signature should be updated to take this in.
         OutputFeatures::default(),
         message_string,
+        // TODO: Add the actual sender signature here.
+        Signature::default(),
+        // TODO:Add the actual script private key here.
+        &Default::default(),
```

Comment:
This might mean the mobile wallet will be unable to spend this, but can be addressed later